src/App.js

```diff
@@ -126,6 +128,16 @@ const App = props => {
               path="/screening/tb/queue"
               redirectOnLogin="/"
             />
+            <Layout
+              component={TbTestQueue}
```

Comment:
@rhenshaw56 .... according to Dave we actually don't need a queue for this station, so this link can be removed... the only way to reach this station will be from the ScreeningList